packages/suite-web/e2e/tests/firmware/firmware.test.ts

```diff
@@ -23,35 +23,43 @@ describe('Firmware', () => {
 
         // check seed screen
         cy.getTestElement('@modal/close-button').should('be.visible'); // modal is cancellable at this moment
-        cy.wait(1000); // wait for animation to finish before taking a screenshot
```

Comment:
not needed, screenshots have been turne off couple of years ago, so why waste time?